wp-content/themes/workingnyc/views/jobs/job.twig

```diff
@@ -11,8 +11,6 @@
       </a>
 
       <p class="c-card__subtitle text-alt">
-        {% if post.sector %}<b>{{ post.sector }}</b>{% endif %}
-        {% if post.sector and post.organization %}<span> {{ __('with', 'WNYC') }} </span>{% endif %}
         {% if post.organization %}<span>{{ post.organization }}</span>{% endif %}
       </p>
     </span>
```

Comment:
There's a section for the summary in this file and job.vue, can you remove it?

Reply:
<div class="c-card__summary">
        <p v-html="post.context.summary"></p>
      </div>

Do I need to remove this div?

Reply:
<div class="c-card__summary">
        <p v-html="post.context.summary"></p>
      </div>

Do I need to remove this div also

Reply:
Yes you should remove the div